service-base-quarkus/src/main/java/org/eclipse/hono/service/quarkus/AbstractProtocolAdapterApplication.java

```diff
@@ -216,10 +224,30 @@ protected CommandRouterClient commandRouterClient() {
      * @return The client.
      */
     protected DeviceConnectionClient deviceConnectionClient() {
-        return new ProtonBasedDeviceConnectionClient(
-                HonoConnection.newConnection(vertx, config.deviceConnection, tracer),
-                messageSamplerFactory,
-                protocolAdapterProperties);
+
+        LOG.debug(deviceConnectionCacheConfig.toString());
+        LOG.debug(config.deviceConnection.toString());
```

Comment:
Was that intended to be kept here? If so, maybe put some more context info in the log output?

Reply:
I have removed the log statements